tests/Dockerfile

```diff
@@ -53,6 +53,7 @@ RUN echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
 RUN echo "config_opts['basedir'] = '/build/mock/'" >> /etc/mock/site-defaults.cfg
 RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpituous\n' >/home/user/.gitconfig && \
     ln -s /cache/images /build/images && \
+    ln -s /cache/images /build/virt-builder && \
```

Comment:
This will throw virt-builder and cockpit images into the same directory. They won't collide on names, but does it really make sense to replicate and distribute them across all of our image stores? Keeping that image around locally seems sufficient.

Are we sure that the virt-builder images never change? I. e. Debian 8 or Ubuntu 16.04 regularly get point releases, and the virt-builder images might get refreshed under the same name. TBH I'd treat them as an implementation detail of virt-builder and not expose them to our mirroring infra.